marshmallow/marshalling.py

```diff
@@ -275,6 +283,22 @@ def deserialize(self, data, fields_dict, many=False, partial=False,
             if value is not missing:
                 key = fields_dict[attr_name].attribute or attr_name
                 set_value(ret, key, value)
+
+        if unknown != IGNORE:
+            for key in set(data) - set(fields_dict):
+                value = data[key]
+                if unknown == ALLOW:
+                    set_value(ret, key, value)
+                elif unknown == RAISE:
+                    def _raise(_):
+                        raise ValidationError('Unkown key.')
```

Comment:
Typo here: "Unknown". Also, we might want to make the error message `'Unknown field.'` to make it more language-agnostic.

Reply:
fixed